tests/array.js

```diff
@@ -91,11 +89,19 @@ describe('Array', () => {
           {
             "a": "a4",
             "b": "b4"
-          }
+          },
+          {
+            "a": 0,
+            "b": undefined
+          },
+          {
+            "a": 1,
+            "b": null
```

Comment:
Testings 0, undefined, and null

Reply:
I am ok with just adding more to this test, but i think the best approach would be to create a specific test just to make sure we handle 0, 1, undefined, null, ... those types that can have some odd behavior